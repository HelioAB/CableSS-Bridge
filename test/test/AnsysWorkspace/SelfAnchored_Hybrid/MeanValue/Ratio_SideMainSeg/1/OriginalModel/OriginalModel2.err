/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       10:59:40
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:03:25
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:08:17
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:18:16
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:19:45
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:19:55
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:21:29
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       11:21:35
/COM,ANSYS RELEASE 2023 R2           BUILD 23.2      UP20230531       15:42:40
