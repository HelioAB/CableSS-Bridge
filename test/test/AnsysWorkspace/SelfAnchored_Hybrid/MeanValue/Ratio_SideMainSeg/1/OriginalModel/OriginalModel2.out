
 Ansys Mechanical Enterprise                       


 *------------------------------------------------------------------*
 |                                                                  |
 |   W E L C O M E   T O   T H E   A N S Y S (R)  P R O G R A M     |
 |                                                                  |
 *------------------------------------------------------------------*




 ***************************************************************
 *         ANSYS MAPDL 2023 R2          LEGAL NOTICES          *
 ***************************************************************
 *                                                             *
 * Copyright 1971-2023 Ansys, Inc.  All rights reserved.       *
 * Unauthorized use, distribution or duplication is            *
 * prohibited.                                                 *
 *                                                             *
 * Ansys is a registered trademark of Ansys, Inc. or its       *
 * subsidiaries in the United States or other countries.       *
 * See the Ansys, Inc. online documentation or the Ansys, Inc. *
 * documentation CD or online help for the complete Legal      *
 * Notice.                                                     *
 *                                                             *
 ***************************************************************
 *                                                             *
 * THIS ANSYS SOFTWARE PRODUCT AND PROGRAM DOCUMENTATION       *
 * INCLUDE TRADE SECRETS AND CONFIDENTIAL AND PROPRIETARY      *
 * PRODUCTS OF ANSYS, INC., ITS SUBSIDIARIES, OR LICENSORS.    *
 * The software products and documentation are furnished by    *
 * Ansys, Inc. or its subsidiaries under a software license    *
 * agreement that contains provisions concerning               *
 * non-disclosure, copying, length and nature of use,          *
 * compliance with exporting laws, warranties, disclaimers,    *
 * limitations of liability, and remedies, and other           *
 * provisions.  The software products and documentation may be *
 * used, disclosed, transferred, or copied only in accordance  *
 * with the terms and conditions of that software license      *
 * agreement.                                                  *
 *                                                             *
 * Ansys, Inc. is a UL registered                              *
 * ISO 9001:2015 company.                                      *
 *                                                             *
 ***************************************************************
 *                                                             *
 * This product is subject to U.S. laws governing export and   *
 * re-export.                                                  *
 *                                                             *
 * For U.S. Government users, except as specifically granted   *
 * by the Ansys, Inc. software license agreement, the use,     *
 * duplication, or disclosure by the United States Government  *
 * is subject to restrictions stated in the Ansys, Inc.        *
 * software license agreement and FAR 12.212 (for non-DOD      *
 * licenses).                                                  *
 *                                                             *
 ***************************************************************

 2023 R2 
     
 Point Releases and Patches installed:   
     
 Discovery 2023 R2   
 Autodyn 2023 R2 
 SpaceClaim 2023 R2  
 CFX (includes CFD-Post) 2023 R2 
 Chemkin 2023 R2 
 EnSight 2023 R2 
 FENSAP-ICE 2023 R2  
 Fluent (includes CFD-Post) 2023 R2  
 Polyflow (includes CFD-Post) 2023 R2
 Forte (includes EnSight) 2023 R2
 TurboGrid 2023 R2   
 Motion 2023 R2  
 Aqwa 2023 R2
 Speos 2023 R2   
 Mechanical Products 2023 R2 
 Material Calibration App 2023 R2
 ACIS Geometry Interface 2023 R2 
 AutoCAD Geometry Interface 2023 R2  
 Catia, Version 4 Geometry Interface 2023 R2 
 Catia, Version 5 Geometry Interface 2023 R2 
 Catia, Version 6 Geometry Interface 2023 R2 
 Creo Elements/Direct Modeling Geometry Interface 2023 R2
 Creo Parametric Geometry Interface 2023 R2  
 Inventor Geometry Interface 2023 R2 
 JTOpen Geometry Interface 2023 R2   
 NX Geometry Interface 2023 R2   
 Parasolid Geometry Interface  2023 R2   
 Solid Edge Geometry Interface 2023 R2   
 SOLIDWORKS Geometry Interface 2023 R2   
 Academic Student 2023 R2


          *****  MAPDL COMMAND LINE ARGUMENTS  *****
  INITIAL JOBNAME              = OriginalModel                                                                                                                                                                                                                                           
  BATCH MODE REQUESTED (-b)    = LIST
  INPUT FILE COPY MODE (-c)    = COPY
  DISTRIBUTED MEMORY PARALLEL REQUESTED
       4 PARALLEL PROCESSES REQUESTED WITH SINGLE THREAD PER PROCESS
    TOTAL OF     4 CORES REQUESTED
  MPI OPTION                   = INTELMPI
  INPUT FILE NAME              = CONIN$
  OUTPUT FILE NAME             = C:\Users\11440\Desktop\usage\Susp V.4\CableSSB-zh\test\test\AnsysWorkspace\SelfAnchored_Hybrid\MeanValue\Ratio_SideMainSeg\1\OriginalModel\OriginalModel2.out
  START-UP FILE MODE           = READ  
  STOP FILE MODE               = READ  
  LANGUAGE                     = en-us               
  DATABASE SIZE REQUESTED (MB) = 1           
  INITIAL DIRECTORY = C:\Users\11440\Desktop\usage\Susp V.4\CableSSB-zh\test\test\AnsysWorkspace\SelfAnchored_Hybrid\MeanValue\Ratio_SideMainSeg\1\OriginalModel

 RELEASE= 2023 R2              BUILD= 23.2      UP20230531   VERSION=WINDOWS x64 
 CURRENT JOBNAME=OriginalModel2  15:42:42  MAY 03, 2024 CP=      0.047


 CLEAR DATABASE AND RERUN START.ANS

 Ansys Mechanical Enterprise                       

 *****  MAPDL SOLUTION ROUTINE  *****

                      L O A D   S T E P   O P T I O N S

   LOAD STEP NUMBER. . . . . . . . . . . . . . . .     1
   TIME AT END OF THE LOAD STEP. . . . . . . . . .  1.0000    
   AUTOMATIC TIME STEPPING . . . . . . . . . . . .    ON
      INITIAL NUMBER OF SUBSTEPS . . . . . . . . .     1
      MAXIMUM NUMBER OF SUBSTEPS . . . . . . . . .   100
      MINIMUM NUMBER OF SUBSTEPS . . . . . . . . .     1
   MAXIMUM NUMBER OF EQUILIBRIUM ITERATIONS. . . .    15
   STEP CHANGE BOUNDARY CONDITIONS . . . . . . . .    NO
   STRESS-STIFFENING . . . . . . . . . . . . . . .    ON
   TERMINATE ANALYSIS IF NOT CONVERGED . . . . . .YES (EXIT)  
   CONVERGENCE CONTROLS. . . . . . . . . . . . . .USE DEFAULTS
   INERTIA LOADS                        X           Y           Z 
      ACEL . . . . . . . . . . . .  0.0000      0.0000      9.8060    
   PRINT OUTPUT CONTROLS . . . . . . . . . . . . .NO PRINTOUT
   DATABASE OUTPUT CONTROLS
      ITEM     FREQUENCY   COMPONENT
       ALL        ALL               

 MAXIMUM NUMBER OF EQUILIBRIUM ITERATIONS HAS BEEN MODIFIED              
  TO BE, NEQIT = 22, BY SOLUTION CONTROL LOGIC.                          

 *** NOTE ***                            CP =       0.141   TIME= 15:42:43
 Predictor is ON by default for structural elements with rotational      
 degrees of freedom.  Use the PRED,OFF command to turn the predictor     
 OFF if it adversely affects the convergence.                            



  *** MASS SUMMARY BY ELEMENT TYPE ***

  TYPE      MASS
     1  0.126162E+08
     2  0.650894E+07
     4  0.900925E+07
     6   376471.    
     7   87448.3    
     8   6162.63    
 AN AVERAGE OF THE X-, Y-, AND Z-DIRECTION MASS TERMS ARE USED
     FORCE CONVERGENCE VALUE  =  0.5311E+09  CRITERION=  0.2655E+07
     MOMENT CONVERGENCE VALUE =  0.4217E-18  CRITERION=  0.3942E+07

  Process memory allocated for solver              =     4.133 MB
  Process memory required for in-core solution     =     4.000 MB
  Process memory required for out-of-core solution =     3.717 MB

 *** NOTE ***                            CP =       0.234   TIME= 15:42:43
 The Distributed Sparse Matrix Solver is currently running in the        
 in-core memory mode.  This memory mode uses the most amount of memory   
 in order to avoid using the hard drive as much as possible, which most  
 often results in the fastest solution time.  This mode is recommended   
 if enough physical memory is present to accommodate all of the solver   
 data.                                                                   
     DISP CONVERGENCE VALUE   =  0.2411      CRITERION=  0.1206E-01
    EQUIL ITER   1 COMPLETED.  NEW TRIANG MATRIX.  MAX DOF INC= -0.2411    
     FORCE CONVERGENCE VALUE  =  0.1266E+11  CRITERION=  0.1232E+08
     MOMENT CONVERGENCE VALUE =  0.5997E+11  CRITERION=  0.1829E+08
     DISP CONVERGENCE VALUE   =  0.6443E-02  CRITERION=  0.1238E-01 <<< CONVERGED
    EQUIL ITER   2 COMPLETED.  NEW TRIANG MATRIX.  MAX DOF INC= -0.6443E-02
     FORCE CONVERGENCE VALUE  =  0.1008E+08  CRITERION=  0.4781E+07
     MOMENT CONVERGENCE VALUE =  0.4701E+08  CRITERION=  0.7097E+07
     DISP CONVERGENCE VALUE   =  0.2647E-03  CRITERION=  0.1239E-01 <<< CONVERGED
    EQUIL ITER   3 COMPLETED.  NEW TRIANG MATRIX.  MAX DOF INC= -0.2647E-03
     FORCE CONVERGENCE VALUE  =  0.2402E+05  CRITERION=  0.4781E+07 <<< CONVERGED
     MOMENT CONVERGENCE VALUE =  0.1043E+06  CRITERION=  0.7097E+07 <<< CONVERGED
    >>> SOLUTION CONVERGED AFTER EQUILIBRIUM ITERATION   3

   *** ELEMENT RESULT CALCULATION TIMES
     TYPE    NUMBER   ENAME      TOTAL CP  AVE CP

        1       460  BEAM188       0.016   0.000034
        2       210  BEAM188       0.016   0.000074
        3        20  BEAM188       0.000   0.000000
        4       250  BEAM188       0.047   0.000188
        5       212  BEAM188       0.031   0.000147
        6       112  LINK10        0.000   0.000000
        7       272  LINK10        0.000   0.000000
        8        44  LINK10        0.016   0.000355

   *** NODAL LOAD CALCULATION TIMES
     TYPE    NUMBER   ENAME      TOTAL CP  AVE CP

        1       460  BEAM188       0.000   0.000000
        2       210  BEAM188       0.000   0.000000
        3        20  BEAM188       0.000   0.000000
        4       250  BEAM188       0.000   0.000000
        5       212  BEAM188       0.000   0.000000
        6       112  LINK10        0.000   0.000000
        7       272  LINK10        0.000   0.000000
        8        44  LINK10        0.000   0.000000
 *** LOAD STEP     1   SUBSTEP     1  COMPLETED.    CUM ITER =      3
 *** TIME =   1.00000         TIME INC =   1.00000    


 *** MAPDL BINARY FILE STATISTICS
  BUFFER SIZE USED= 16384
        0.062 MB WRITTEN ON ELEMENT MATRIX FILE: OriginalModel2.emat
        3.125 MB WRITTEN ON ELEMENT SAVED DATA FILE: OriginalModel2.esav
        0.875 MB WRITTEN ON ASSEMBLED MATRIX FILE: OriginalModel2.full
        0.812 MB WRITTEN ON RESULTS FILE: OriginalModel2.rst

 FINISH SOLUTION PROCESSING


 ***** ROUTINE COMPLETED *****  CP =         0.453



 ***** END OF INPUT ENCOUNTERED *****


 NUMBER OF WARNING MESSAGES ENCOUNTERED=          0
 NUMBER OF ERROR   MESSAGES ENCOUNTERED=          0


 *-----------------------------------------------------------------------------*
 |                                                                             |
 |                               RUN COMPLETED                                 |
 |                                                                             |
 |-----------------------------------------------------------------------------|
 |                                                                             |
 |  Ansys MAPDL 2023 R2         Build 23.2         UP20230531    WINDOWS x64   |
 |                                                                             |
 |-----------------------------------------------------------------------------|
 |                                                                             |
 |  Database Requested(-db)     1 MB     Scratch Memory Requested       240 MB |
 |  Maximum Database Used       1 MB     Maximum Scratch Memory Used    135 MB |
 |                                                                             |
 |-----------------------------------------------------------------------------|
 |                                                                             |
 |        CP Time      (sec) =          0.453       Time  =  15:42:44          |
 |        Elapsed Time (sec) =          5.000       Date  =  05/03/2024        |
 |                                                                             |
 *-----------------------------------------------------------------------------*
